docs/querying/sql.md

```diff
@@ -1108,6 +1108,18 @@ GROUP BY 1
 ORDER BY 2 DESC
 ```
 
+If you want to retrieve segment that was compacted (ANY compaction):
+
+```sql
+SELECT * FROM sys.segments WHERE last_compaction_state is not null
+```
+
+or if you want to retrieve segment that was compacted only by a particular compaction spec (such as that of the auto compaction):
```

Comment:
```suggestion
Sometimes, you may have updated your compaction spec to a more optimal spec. In these cases you can find which segments are using the latest compaction spec using a query like the one below:
```

Can you also add instructions on how to produce the `last_compaction_state` blob from the web console. If I'm looking at the compaction config for a datasource - can I copy something to paste in the where clause here?

Reply:
Done